src/target/riscv_debug.c

```diff
@@ -965,6 +1004,45 @@ static const char *riscv_fpu_ext_string(const uint32_t extensions)
  * Both are then continued with:
  * "		<reg name=\"pc\" bitsize=\"[address_width]\" type=\"code_ptr\"/>"
  * "	</feature>"
+ * The following are only generated for an F core (single precision HW float support):
```

Comment:
We would move this comment block to the front of the FPU block function so the documentation for it is kept together with the function.